configs/wyoming.nrel-op.json

```diff
@@ -44,6 +44,7 @@
         "trip_end_notification": false
     },
     "admin_dashboard": {
+        "admin_access": "nschultz@nrel.gov",
```

Comment:
Please revert this change since we don't actually want to give you access to the wyoming data

Reply:
Reverted :)